### PR TITLE
Fix typos in metric type definition

### PR DESCRIPTION
**Pull request by kalantar**
Fixes issue https://github.com/iter8-tools/etc3/issues/160

### CODE REVIEW

File: api/v2alpha2/metric_types.go [hunk 1]
```diff
@@ -68,14 +68,14 @@ type MetricSpec struct {
...
-	SecretRef *string `json:"secret,omitempty" yaml:"secret,omitempty"`
+	Secret *string `json:"secret,omitempty" yaml:"secret,omitempty"`
 
 	// HeaderTemplates are templates for headers that should be passed to the metrics backend.
 	// Typically these are authentication headers. Any fields of the form '$name' are treated as
 	// variables whose value is looked up in the secret referred to by spec.secretRef. If the
 	// secret does not contain the variable as a key, the full string "$name" is used.
 	// +optional
-	HeaderTemplates *[]NamedValue `json:"headers,omitempty" yaml:"headers,omitempty"`
+	HeaderTemplates *[]NamedValue `json:"headerTenplates,omitempty" yaml:"headers,omitempty"`
```
Comment (sriumcp) [inline api/v2alpha2/metric_types.go:78]: Please fix two types here: `headerTemplates` (m instead of n) and `headerTemplates` (instead of headers)

**Review by sriumcp — CHANGES_REQUESTED** [review]
typos

File: api/v2alpha2/metric_types.go (hunk 1 above)
Comment (kalantar) [reply api/v2alpha2/metric_types.go]: done

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**